src/Serilog/Context/LogContext.cs

```diff
@@ -42,11 +49,19 @@ namespace Serilog.Context
     /// </code>
     /// </example>
     /// <remarks>The scope of the context is the current logical thread, using
-    /// <see cref="CallContext.LogicalGetData"/> (and so is
-    /// preserved across async/await calls).</remarks>
+#if ASYNCLOCAL
+    /// <seealso cref="AsyncLocal{T}"/>
+#else
+    /// <seealso cref="CallContext"/>
```

Comment:
Can we add the `.LogicalGetData` so that it's easier to track down docs on MSDN?